rust/datafusion/src/execution/context.rs

```diff
@@ -450,6 +451,10 @@ impl ExecutionContext {
                 op.clone(),
                 self.create_physical_expr(right, input_schema)?,
             ))),
+            Expr::Length(expr) => Ok(Arc::new(Length::try_new(
```

Comment:
I wonder if we really want to add a new `Expr` specifically for this function when we already have way to register UDFs. It seems like it might be better to treat it just like the other UDFs?